person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] <b>OPTIONAL:</b> It would be better to use `Id` as random `@id = Random.rand(1..1000)`.

Reply:
Thank you, I will add your suggestion 😃